lib/state/StateDevice.js

```diff
@@ -45,9 +45,19 @@ class Device {
     }
   }
 
-	/**
+  /**
+   * Adds a cookie to the device. Cookies can contain any information usefull to the integration
+   * @param {Map} cookie
```

Comment:
I don't think we limit the cookie type to be a map, although it probably will a map in most cases

Reply:
:thumbsup: